codefori/vscode-ibmi
# Fix/namespace cleanup

### Changes

Change logic of the connection function to separate the UI and API.

### How to test this PR

Examples:

1. Run the test cases
2. Ensure the user can connect

### Checklist
